src/content/developers/docs/accounts/index.md

```diff
@@ -25,7 +25,7 @@ Both account types have the ability to:
 
 ### Key differences {#key-differences}
 
-**Externally-owned**
+**Externally-owned (wallet address)**
```

Comment:
I dont think this is quite right. There are smart contract wallets which would not be externally owned accounts.